test/mitmproxy/contentviews/image/test_view.py

```diff
@@ -9,8 +9,7 @@ def test_view_image():
         "mitmproxy/data/image.png",
         "mitmproxy/data/image.gif",
         "mitmproxy/data/all.jpeg",
-        # https://bugs.python.org/issue21574
-        # "mitmproxy/data/image.ico",
+        "mitmproxy/data/image.ico",
```

Comment:
Nope, it is about porting image type detections from PIL to imghdr. I add the test for ico to imghdr.tests so we can detect .ico from imghdr.what